rs/tests/testnets/BUILD.bazel

```
@@ -279,6 +279,34 @@ system_test_nns(
     ],
 )
 
+system_test_nns(
+    name = "perf_benchmark",
```

What about making it explicit this is about **I/O** performance testing by renaming this to:
```suggestion
    name = "io_perf_benchmark",
```